### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by aidanraskil**
This is a new features

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (iskandarali) [inline index.php:1]: Write a proper code

Comment (aidanraskil) [inline index.php:1]: what?

**Review by aidanraskil — COMMENTED** [review]
Is

**Review by iskandarali — COMMENTED** [review]
How

**Comment by iskandarali** [comment]
Bullshit

**Review by iskandarali — COMMENTED** [review]
ha

**Pull request closed without merging**